ajax/libs/weblas/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "weblas",
+  "description": "GPU accelerated BLAS for node and the browser",
+  "filename": "weblas.min.js",
+  "keywords": [
+    "gpu",
+    "blas",
+    "gemm",
+    "matrix",
+    "multiply",
+    "neural network",
+    "deep learning"
+  ],
+  "author": "Waylon Flinn <waylonflinn@gmail.com>",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/waylonflinn/weblas"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/waylonflinn/weblas.git",
```

Comment:
Let's use npm auto-update because the latest version is from npm.
```
  "npmName": "weblas",
  "npmFileMap": [
    {
      "basePath": "dist",
      "files": [
        "**/!(*.json)"
      ]
    }
  ],
```
However, would you please add the old versions from GitHub?
Many thanks.